content/swiftui/concepts/widgets/widgets.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'Widgets'
+Description: 'Building user interfaces across Apple platforms.'
+Subjects: 
+    - 'iOS Development'
+Tags: 
+    - 'SwiftUI'
+    - 'User Interface'
+    - 'iOS'
+CatalogContent: 
+    - 'paths/ios-developer'
+    - 'learn-swift'
+---
+
+# SwiftUI Widgets
+
+## What are SwiftUI Widgets?
+**Widgets** in SwiftUI  are interactive components that provide users with glanceable information or quick access to app functionality directly from the home screen of their device. They are mini-applications that can display dynamic content, such as weather updates, calendar events, or news headlines. SwiftUI widgets can be created using the WidgetKit framework and are highly customizable, allowing developers to design visually appealing and personalized experiences for users without requiring them to open the full app.
+
+## To create a widget using the Widget Extension template in Xcode, follow these steps:
+
+1. Open Xcode and create a new project or open an existing project.
+2. In the project navigator, right-click on the project's root folder.
+3. Select "New File" from the contextual menu.
+4. In the template selection dialog, search for "Widget Extension" or       navigate to "App Extension" -> "Widget Extension."
+5. Click "Next" to proceed.
+6. Provide a name for your widget extension and choose the target app.
+7. Select the desired options such as widget size and intent configuration (if applicable).
+8. Click "Finish" to create the widget extension.
+
+Xcode will generate the necessary files and folders for the widget extension, including a default SwiftUI widget view and a widget extension entry point.
+
+Next, you can customize your widget by modifying the SwiftUI view and implementing the widget's data provider.
+
+In the generated widget view file (e.g., YourWidget.swift), you can modify the appearance and content of the widget by editing the body property. Add SwiftUI components, apply styles, and display dynamic data to create your desired widget UI.
+
+In the generated widget extension entry point file (e.g., YourWidgetExtension.swift), you'll find a struct conforming to Widget protocol. You can modify this struct to configure your widget's behavior, supported families, and timeline provider.
+
+After customizing the widget view and the widget extension, you can build and run your app to see the widget in the widget gallery.
+
+To add your widget to the home screen, long-press on the home screen, tap the "+" button, search for your app's name, and select your widget to add it to the desired location.
+
+Remember to adjust the code and configurations as per your project's requirements to create a fully functional and visually appealing SwiftUI widget.
```

Comment:
[Nitpick] Pronouns are not allowed to be used according to the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#writing-entries), so consider revising. 
```suggestion
In the generated widget view file (e.g., YourWidget.swift), you can modify the appearance and content of the widget by editing the body property. Add SwiftUI components, apply styles, and display dynamic data to create your desired widget UI.

In the generated widget extension entry point file (e.g., YourWidgetExtension.swift), you'll find a struct conforming to Widget protocol. You can modify this struct to configure your widget's behavior, supported families, and timeline provider.

After customizing the widget view and the widget extension, you can build and run your app to see the widget in the widget gallery.

To add your widget to the home screen, long-press on the home screen, tap the "+" button, search for your app's name, and select your widget to add it to the desired location.

Remember to adjust the code and configurations as per your project's requirements to create a fully functional and visually appealing SwiftUI widget.
```

Reply:
Let's remove pronouns here (I, we, you, etc.)